lib/js/components/Modals/ModalDialog/ModalDialog.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+	<div class="modalDialog">
```

Comment:
do we need this wrapper?

Reply:
[`3241908` (#248)](https://github.com/bethinkpl/design-system/pull/248/commits/3241908232391d94d55d8a61fc9114470b47dcbf)